tools/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -822,7 +823,7 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
         assert(appInfo.nonEmpty)
         assert(headers.size ==
           QualOutputWriter.getSummaryHeaderStringsAndSizes(Seq(appInfo.get), 0).keys.size)
-        assert(values.size == headers.size)
+        assert(values.size == headers.size - 2) // UnsupportedExec and UnsupportedExprs are empty
```

Comment:
do we have any tests that verify we get the unsupported string back?

Reply:
I have verified it locally. Will add tests for both unsupportedExec and UnsupportedExprs

Reply:
Added test which verifies unsupported execs and expressions